index.html

```diff
@@ -88,25 +88,28 @@ <h2>A project for first-time contributions</h2>
 
         <!-- ________ *Your_Name* card START ________  -->
         <div class="card">
-          <p class="name">Your name</p>
+          <p class="name">Joseph adeniji</p>
           <p class="contact">
             <i class="fab fa-x-twitter"></i>
-            <a href="https://www.twitter.com/your_user_handle" target="_blank">Your handle</a>
+            <a href="https://twitter.com/joshiz02" target="_blank">Your handle</a>
             <i class="fab fa-github"></i>
-            <a href="https://www.github.com/your_user_handle" target="_blank">Your handle</a>
+            <a href="https://github.com/jophizz" target="_blank">Your handle</a>
           </p>
-          <p class="about">about you</p>
+          <p class="i'm a full stack developer and looking to contribute to an open-source project"></p>
           <div class="resources">
             <p>3 Useful Dev Resources</p>
             <ul>
               <li>
-                <a href="#" target="_blank" title="First Resource">Resource 1</a>
+                <a href="https://stackoverflow.com" target="_blank" title="First Resource">Resource 1</a>
+                answer to technical questions
               </li>
               <li>
-                <a href="#" target="_blank" title="Second Resource">Resource 2</a>
+                <a href="https://developer.mozilla.org" target="_blank" title="Second Resource">Resource 2</a>
+                MDN documentation
               </li>
               <li>
-                <a href="#" target="_blank" title="Third resource">Resource 3</a>
+                <a href="https://www.ibm.com" target="_blank" title="Third resource">Resource 3</a>
+                cyber security news
               </li>
             </ul>
           </div>
```

Comment:
Do not make any changes on these lines, as these lines are meant to be the reference for new contributors. Seems like you have modified these lines by updating your details here. Kindly undo those changes.